app/backend/src/data/dummy_communities.json

```diff
@@ -0,0 +1,219 @@
+{
+  "nodes": [
+    {
+      "name": "World",
+      "admins": ["aapeli", "itsi", "lucas", "val"],
+      "members": ["kellyt", "misspoptart", "debbie", "joshua", "chagai"],
+      "parent": null,
+      "coordinates": [
+        [
+          -180.0,
+          -90
+        ],
+        [
+          180.0,
+          -90
+        ],
+        [
+          180,
+          90
+        ],
+        [
+          -180.0,
+          90
+        ],
+        [
+          -180.0,
+          -90
+        ]
+      ]
+    },
+    {
+      "name": "United States",
+      "admins": ["aapeli", "val"],
+      "members": ["misspoptart", "joshua"],
+      "parent": "World",
+      "osm_id": 148838,
```

Comment:
Are these just for dummy data? They could do with cleaning up if not.

Reply:
For dummy data only. I haven't explained in comments, but I actually simplify the geometries a fair bit in the code (it was too large/too slow without it). This is the `geom_simplify` field.